app/Http/Controllers/Admin/CourseController.php

```diff
@@ -38,4 +39,26 @@ public function show($id)
 
         return view('students.index', $data);
     }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param \App\Http\Requests\Course\UpdateRequest $request
+     * @param int                                     $id
+     *
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function update(UpdateRequest $request, $id)
+    {
+        DB::transaction(function () use ($request, $id) {
+            $course = Course::findOrFail($id);
+
+            $course->fill($request->all());
```

Comment:
Another suggestion of the same nature... Mostly to reduce the length of the code :smile: 

Instead of these:
```php
$course->fill($request->all());
$course->save();
```
To this:
```php
$course = tap($course)->update($request->all());
```